Makefile

```diff
@@ -26,31 +26,39 @@ docker-push-e2e-test:
 build-docker-compose-tests:
 	docker-compose -f integration-tests/docker-compose-configs/docker-compose.base.yaml build --no-cache
 
-run-e2e-test-all:
-	make run-e2e-test-default-config
-	make run-e2e-test-allowlist
-	make run-e2e-test-unique-address
-	make run-e2e-test-sep24-withdrawal
+run-integration-test-all:
```

Comment:
Thanks for renaming the targets. After merging this, please make sure the e2e tests work in the CI pipeline.